ocw_import/api.py

```diff
@@ -67,19 +81,19 @@ def import_ocw2hugo_content(bucket, prefix, website):  # pylint:disable=too-many
             parent = None
             if len(s3_content_parts) >= 1:
                 content_json = yaml.load(s3_content_parts[0], Loader=yaml.Loader)
+                layout = content_json.get("layout", None)
                 menu = content_json.get("menu", None)
+                uuid = content_json.get("uid", None)
```

Comment:
I should be able answer this myself once I can complete a successful run, but do you know if `menu_values.get("identifier")` will ever differ from `content_json.get("uid")`?  If so, we might end up with dupes unless the old imports from previous runs of this command are deleted first.

Reply:
They are pulled from the same property, the only difference I can think of might possibly be course home pages with no uid property, they would maybe be undefined but I'll have to check on that.